src/hstr.c

```diff
@@ -328,73 +329,53 @@ unsigned recalculate_max_history_items()
 
 void hstr_get_env_configuration(Hstr *hstr)
 {
-    char *hstr_config=getenv(HH_ENV_VAR_CONFIG);
-    if(hstr_config && strlen(hstr_config)>0) {
-        if(strstr(hstr_config,HH_CONFIG_THEME_MONOCHROMATIC)) {
+    const char *hstr_config=getenv(HH_ENV_VAR_CONFIG);
+    if (!hstr_config || !strlen(hstr_config)) {
+        return;
+    }
+    size_t const len = strlen(hstr_config)+1;
+    char* config_items = malloc(len);
+    memcpy(config_items, hstr_config, len);
+    char* saveptr = config_items;
+    char* item;
+    while ((item = strtok_r(saveptr, ",", &saveptr)) != NULL) {
```

Comment:
Unfortunately refactoring of if statements in this loop is wrong - there is a semantics in which option can overwrite other and making this flat kills the logic that was implemented in there. Thus I cannot merge this particular change.